src/fu-device-list.c

```diff
@@ -115,6 +115,19 @@ fu_device_list_get_all (FuDeviceList *self)
 	return devices;
 }
 
+static gint
+fu_device_list_sort_by_priority (gconstpointer a, gconstpointer b)
+{
+	gint first = fu_device_get_priority (*(FuDevice **) a);
+	gint second = fu_device_get_priority (*(FuDevice **) b);
+
+	if (first < second)
+		return 1;
+	if (first == second)
```

Comment:
In other parts of the code we;ve got a check for `<`, then `>`, then falling back to 0 -- it might be nice to match.